test/NuGet.Core.FuncTests/NuGet.Packaging.FuncTest/SigningTests/SignatureTrustAndValidityVerificationProviderTests.cs

```diff
@@ -130,7 +130,7 @@ public async Task VerifySignaturesAsync_ValidCertificateAndTimestampWithDifferen
             }
         }
 
-        [PlatformFact(Platform.Windows, Platform.Darwin)] // https://github.com/NuGet/Home/issues/9771
+        [CIOnlyFact]
```

Comment:
What's the reason we don't want to run the test locally?

Reply:
Sorry, I'm not sure about the reason. And I didn't find anything from the previous PR adding this change.
Those tests need admin privilege if running locally, or else, they will fail. I'm not sure if this is the reason.
Hi @dtivel , do you have any ideas why we don't want to run those tests locally? Thanks.